crates/weaver_semconv/data/event.yaml

```diff
@@ -0,0 +1,181 @@
+groups:
+  - id: device.app.lifecycle
+    stability: experimental
+    type: event
+    name: device.app.lifecycle
+    brief: >
+      This event represents an occurrence of a lifecycle transition on Android or iOS platform.
+    note: >
+      This event identifies the fields that are common to all lifecycle events for android and iOS using
+      the `android.state` and `ios.state` fields. The `android.state` and `ios.state` attributes are
+      mutually exclusive.
+    body:
+      id: device_lifecycle
```

Comment:
The `id` is now (currently) required at this location (because we want the `fields` to have an `id`)

Reply:
What's the difference between "id" and "name"?

Let's be explicit in the comments which can be unique and how you reference something.

Reply:
Based on how weaver handles the `id` for everything else the `id` in the yaml becomes the `name` in the resolved definition.... Sooo. they are the same.

Reply:
gotcha - Sorry I missed that.